regulations/uitests/definition_test.py

```diff
@@ -77,20 +78,19 @@ def test_definition(self):
         # go to 1005-1-a
         toc_toggle.click()
         WebDriverWait(self.driver, 10)
-        self.driver.execute_script(
-            "window.scrollTo(0, document.body.scrollHeight);")
+        self.driver.execute_script('window.scrollTo(0, 5);')
```

Comment:
I think the original test was [modified](https://github.com/eregs/regulations-site/commit/1a45058d5d49c5f3b3a5d5befd100c970053b737) into a no-longer useful state. Let's try to bring back it's intent (verifying that the header matches the scroll position). That was why my [patches](https://github.com/eregs/regulations-site/compare/master...cmc333333:sauce-fail) scrolled through the doc and created a set of headers.